scrapper.py

```diff
@@ -25,30 +30,52 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls: list, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.max_articles_per_seed = max_articles_per_seed
```

Comment:
you can skip this param for now and just go ahead and implement crawler logic itself. After completing crawler - you may add this param.